app/konnect/analytics/services-and-routes.md

```diff
@@ -3,9 +3,8 @@ title: Analyze Services and Routes
 content_type: how-to
 ---
 
-<!-- this page is deprecated-->
 
-In the Service Hub, the service, service version, and route graphs provide dynamic
+In the API product builder the API product, API product version, and route graphs provide dynamic
```

Comment:
We don't currently have a dedicated API product version chart -- just a chart showing traffic broken down by version for a given API product.  (We do have a route chart.)